src/main/java/com/derongan/minecraft/deeperworld/listeners/MovementListener.kt

```diff
@@ -54,9 +54,10 @@ object MovementListener : Listener {
     private fun onPlayerMoveInternal(player: Player, from: Location, to: Location) {
         val current = WorldManager.getSectionFor(player.location) ?: let {
             //damage players outside of sections
-            if (!DeeperConfig.data.damageExcludedWorlds.contains(player.location.world)
-                && DeeperConfig.data.damageOutsideSections > 0.0
+            if (DeeperConfig.data.damageOutsideSections > 0.0
+                && !DeeperConfig.data.damageExcludedWorlds.contains(player.location.world)
```

Comment:
What i meant by the contains thing is:

```suggestion
                && player.location.world !in DeeperConfig.data.damageExcludedWorlds
```